Code/GraphMol/MolAlign/Wrap/rdMolAlign.cpp

```diff
@@ -197,6 +198,39 @@ double AlignMolecule(ROMol &prbMol, const ROMol &refMol, int prbCid = -1,
   return rmsd;
 }
 
+double getConformerRMS(ROMol &mol, int confId1, int confId2,
+                       python::object atomIds = python::list(),
+                       bool prealigned = false) {
+  double rmsd;
+  if (atomIds) {
+    std::vector<unsigned int> *aIds = _translateIds(atomIds);
+    rmsd = MolAlign::getConformerRMS(mol, confId1, confId2, aIds, prealigned);
+    delete aIds;
+  } else {
+    rmsd = MolAlign::getConformerRMS(mol, confId1, confId2, 0, prealigned);
+  }
+  return rmsd;
+}
+
+double getBestRMS(const ROMol &ref, ROMol &probe, int refConfId,
+                  int probeConfId, python::object maps = python::list()) {
+  double rmsd;
+  if (maps) {
+    std::vector<MatchVectType> atomMaps;
+    //    BOOST_FOREACH (const python::object &map, *maps) {
+    //      MatchVectType *atomMap = _translateAtomMap(map);
```

Comment:
You'll need to use the magic of python::extract.  You'll probably also need to make tuple and list versions, I don't know how to do sequences yet.

It will be something like:

```
template<class T, class U>
std::vector<std::vector<U>> ConvertVectVect(T vectvect) {
  std::vector<std::vector<U> > vect;
  size_t size = python::extract<unsigned int>(vectvect.attr("__len__")());
  vect.resize(size);
  for(size_t i=0; i<size; ++i) {
    unsigned int len1 = python::extract<unsigned int>(vectvect[i].attr("__len__")());
    std::vector<U> &v = vect[i];
    v.reserve(len1);
    for(unsigned int j=0;j<len1;++j){
      U u = python::extract<U>(vectvect[i][j]);
      v.push_back(u);
    }
  }
  return vect;
}
```

with usage here:

```
double getBestRMS(const ROMol &ref, ROMol &probe, int refConfId,
                                  int probeConfId, python::list maps) {
...
std::vector<MatchVectType> atomMaps = ConvertVectVec<python::list, MatchType>(maps);
...
}

double getBestRMS(const ROMol &ref, ROMol &probe, int refConfId,
                                  int probeConfId, python::tuple maps) {
...
std::vector<MatchVectType> atomMaps = ConvertVectVec<python::tuple, MatchType>(maps);
...
}
```

I would put initialize the default args in the wrapped arguments.